smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/rendering/endpoints/EndpointParametersGenerator.kt

```diff
@@ -38,6 +38,8 @@ class EndpointParametersGenerator(
                 name = CLASS_NAME
                 namespace = "${settings.pkg.name}.endpoints"
             }
+
+        fun getParamKotlinName(param: Parameter): String = param.name.toKotlin()
     }
 
     private val params: List<KotlinEndpointParameter> = rules.parameters.toList()
```

Comment:
GH won't let me comment on the unchanged line but we generate a companion object for `EndpointParameters` like this:

```kotlin
    public companion object {
        public operator fun invoke(block: Builder.() -> Unit): EndpointParameters = Builder().apply(block).build()
    }
```

This function should probably be marked `inline` to avoid unnecessary backing lambdas everywhere we have to construct endpoint parameters in generated code (contributes to bloated artifact size)